utils/githooks/pre-commit.d/71-flake.sh

```diff
@@ -20,19 +20,17 @@
 set -ue
 
 _print_githook_header "Flake8"
-if ! command -v flake8 > /dev/null 2>&1; then
-    echo "flake8 not installed. Install flake8 command to improve pre-commit checks:"
-    echo "  python3 -m pip install -r ./utils/cq/requirements.txt"
-    exit 0
-fi
 
-if flake8 --version | grep ^6\\.; then
-    echo "flake8 >= 6.x does not have the --diff option. Skipping."
```

Comment:
This check is needed for platforms not managing the CQ dependencies with `pip`, that might be stuck at 6.0.0.

Reply:
I specifically did not want to break backwards compat so I verified the hook still works with python 3.6 + flake<6 (what we use today) and python3.9 + flake 7.1.1 (what this PR updates to)